pyg_lib/csrc/sampler/cpu/random_walk_kernel.cpp

```diff
@@ -32,7 +32,7 @@ torch::Tensor random_walk_kernel(const torch::Tensor& rowptr,
         for (auto j = 1; j < walk_length + 1; ++j) {
           scalar_t row_start = rowptr_data[v], row_end = rowptr_data[v + 1];
           if (row_end - row_start > 0)
-            v = col_data[randint<scalar_t>(row_start, row_end)];
+            v = col_data[randint(row_start, row_end)];
```

Comment:
Interesting that this does work without it.

Reply:
Function templates can induce the types by examining args, making coding easier :)